### PR TITLE
Fix AppManifest path

### PR DESCRIPTION
**Pull request by doosuu**
# Description

# Description

Fixes the path to the AppManifest used in the Container Build workflow.

## Azure DevOps PBI/Task reference

-

## Checklist

* [ ] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [ ] Vehicle App can process MQTT messages and call the seat service
* [ ] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully

* [ ] Release workflow is passing

### CODE REVIEW

File: NOTICE-3RD-PARTY-CONTENT.md [hunk 1]
```diff
@@ -40,21 +40,17 @@
...
-|actions/checkout|v2|MIT License|
 |actions/checkout|v3|MIT License|
 |actions/upload-artifact|v2|MIT License|
 |actions/upload-artifact|v3.1.0|MIT License|
 |aquasecurity/trivy-action|0.3.0|Apache License 2.0|
 |ASzc/change-string-case-action|v1|ISC License|
 |docker/build-push-action|v2|Apache License 2.0|
 |docker/login-action|v1|Apache License 2.0|
-|docker/metadata-action|98669ae865ea3cffbcbaa878cf57c20bbf1c6c38|Apache License 2.0|
 |docker/setup-buildx-action|v1|Apache License 2.0|
```
Comment (mml5bg) [inline NOTICE-3RD-PARTY-CONTENT.md:50]: Please increase version to v2, in v2 Node16 is introduced as default runtime

File: NOTICE-3RD-PARTY-CONTENT.md [hunk 2]
```diff
@@ -40,21 +40,17 @@
...
 |docker/setup-qemu-action|v1|Apache License 2.0|
```
Comment (mml5bg) [inline NOTICE-3RD-PARTY-CONTENT.md:51]: Please increase version to v2, in v2 Node16 is introduced as default runtime

File: NOTICE-3RD-PARTY-CONTENT.md [hunk 3]
```diff
@@ -40,21 +40,17 @@
 ## Workflows
 | Dependency | Version | License |
 |:-----------|:-------:|--------:|
-|actions/checkout|v2|MIT License|
 |actions/checkout|v3|MIT License|
 |actions/upload-artifact|v2|MIT License|
 |actions/upload-artifact|v3.1.0|MIT License|
```
Comment (mml5bg) [inline NOTICE-3RD-PARTY-CONTENT.md:45]: Why you use both V2 and v3.1.0, I would suggest to use only v3.1.0

File: NOTICE-3RD-PARTY-CONTENT.md [hunk 4]
```diff
@@ -40,21 +40,17 @@
...
 |aquasecurity/trivy-action|0.3.0|Apache License 2.0|
 |ASzc/change-string-case-action|v1|ISC License|
 |docker/build-push-action|v2|Apache License 2.0|
 |docker/login-action|v1|Apache License 2.0|
```
Comment (mml5bg) [inline NOTICE-3RD-PARTY-CONTENT.md:49]: Please increase version to v2, in v2 Node16 is introduced as default runtime

**Review by mml5bg — COMMENTED** [review]
Please take a look on review comments

File: NOTICE-3RD-PARTY-CONTENT.md (hunk 4 above)
Comment (doosuu) [reply NOTICE-3RD-PARTY-CONTENT.md:49]: Will be done in the next PR

File: NOTICE-3RD-PARTY-CONTENT.md (hunk 3 above)
Comment (doosuu) [reply NOTICE-3RD-PARTY-CONTENT.md:45]: Will be done in the next PR

File: NOTICE-3RD-PARTY-CONTENT.md (hunk 2 above)
Comment (doosuu) [reply NOTICE-3RD-PARTY-CONTENT.md:51]: Will be done in the next PR

File: NOTICE-3RD-PARTY-CONTENT.md (hunk 1 above)
Comment (doosuu) [reply NOTICE-3RD-PARTY-CONTENT.md:50]: Will be done in the next PR

**Review by mml5bg — APPROVED** [review]
LGTM

**Pull request merged**